bundles/org.pitest.pitclipse.ui/src/org/pitest/pitclipse/ui/view/PitView.java

```diff
@@ -44,17 +84,38 @@ public synchronized void createPartControl(Composite parent) {
         }
     }
 
+    /**
+     * Set URL to blank and reset homeUrl, to have a clean browser after
+     * creation<br>
+     * <b>Needed</b> for testing purposes
+     */
+    private void resetBrowser() {
+        browser.setUrl(BLANK_PAGE);
+        homeUrlString = null;
+    }
+
     @Override
     public void setFocus() {
-        // Nothing to do
+        if (browser != null && !browser.isDisposed()) {
```

Comment:
If understand correctly (we've already talked about that a few days ago), unless something really bad happened, the browser cannot be null, so I'd avoid such a check, which is also uncovered (https://sonarcloud.io/component_measures?id=org.pitest%3Aorg.pitest.pitclipse&metric=new_coverage&pullRequest=169&selected=org.pitest%3Aorg.pitest.pitclipse%3Abundles%2Forg.pitest.pitclipse.ui%2Fsrc%2Forg%2Fpitest%2Fpitclipse%2Fui%2Fview%2FPitView.java&view=list)

while I guess the browser can already be disposed? I see that condition seems to be covered...